pkg/kubelet/kubelet.go

```diff
@@ -757,6 +758,22 @@ func NewMainKubelet(kubeCfg *kubeletconfiginternal.KubeletConfiguration,
 	}
 	klet.imageManager = imageManager
 
+	if containerRuntime == kubetypes.RemoteContainerRuntime && utilfeature.DefaultFeatureGate.Enabled(features.CRIContainerLogRotation) {
+		maxSize, err := logs.ParseMaxSize(kubeCfg.ContainerLogMaxSize)
```

Comment:
I think it would be nice to keep this logic in the log container manager.  You could just pass both kubeCfg options to `NewContainerLogManager`, and do the parsing inside that function.  Then there is no need to make `ParseMaxSize` an exported function.

Reply:
Copied from eviction manager. https://github.com/Random-Liu/kubernetes/blob/e0b0dbf06b9bd7eb0e972946673583216e0a7016/pkg/kubelet/kubelet.go#L436

Reply:
I can move it into `NewContainerLogMnager`. :) I actually prefer that.

Reply:
I think the eviction thresholds are used by the container manager as well?

Reply:
Done